src/components/Tabs/Tabs.js

```diff
@@ -1,67 +1,74 @@
-import * as PropTypes from 'prop-types';
 import * as React from 'react';
+import * as PropTypes from 'prop-types';
 import { uuid4 } from '../../utils/utils';
 import './Tabs.css';
 
-let uuid;
-const tabsToRender = [];
-const tabsContentToRender = [];
-
 /* eslint-disable */
 /**
  * @uxpincomponent
  * @uxpinwrappers
  * SkipContainerWrapper
  */
-export default function Tabs(props) {
-  uuid = uuid4();
-  tabsToRender.length = 0;
-  tabsContentToRender.length = 0;
-
-  Array(11).fill()
-    .forEach((_, i) => {
-      if (props[`tab${i}`]) {
-        tabsToRender.push(
-          <li
-            key={`key${uuid}${i}`}
-            className={`${props.activeTab === i ? '-active' : ''} -d--flex`}
-            onClick={() => {
-              if (props[`click${i}`]) {
-                props[`click${i}`]();
-              }
-            }}>
-            <a
-              href={`#tab${uuid}${i}`}>
-              {props[`tab${i}`]}
-            </a>
-          </li>
-        );
-        tabsContentToRender.push(
-          <div key={`content${uuid}${i}`} className={`chi-tabs-panel ${props.activeTab === i ? '-active' : ''}`} id={`tab${uuid}${i}`}></div>
-        );
-      }
-    });
+export default class Tabs extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { id: uuid4() };
+  }
 
+  componentDidMount() {
     const initialize = setInterval(() => {
-      if (window.chi && document.getElementById(uuid)) {
-        window.chi.tab(document.getElementById(uuid));
+      if (window.chi && document.getElementById(this.state.id)) {
+        window.chi.tab(document.getElementById(this.state.id));
         clearInterval(initialize);
       }
-    }, 100);
+    }, 1000);
```

Comment:
Based on the `interval` nature of this check, is increasing the interval necessary here?